zappa/cli.py

```diff
@@ -2587,6 +2587,12 @@ def get_zappa_settings_string(self):
             arn = event.get("event_source", {}).get("arn")
             function = event.get("function")
             if arn and function:
+                if arn.find('aws:kafka') > 0:
+                    # only support one kafka topic
+                    topics = event.get("event_source", {}).get("topics")
+                    if isinstance(topics, list):
+                        topics = topics[0]
```

Comment:
if only 1 topic, change variable name to `topic`.

Reply:
fixed